tesseract/tesseract_planning/tesseract_motion_planners/test/serialize_test.cpp

```diff
@@ -45,164 +45,175 @@ TESSERACT_COMMON_IGNORE_WARNINGS_POP
-  EXPECT_TRUE(toXMLFile(plan_profile, tesseract_common::getTempPath() + "descartes_default_plan_example_input.xml"));
+  DescartesDefaultPlanProfile<double>::Ptr plan_profile = genDescartesPlanProfile();
+  EXPECT_TRUE(toXMLFile(*plan_profile, tesseract_common::getTempPath() + "descartes_default_plan_example_input.xml"));
 
   // Import file
-  DescartesDefaultPlanProfileD imported_plan_profile = descartesPlanFromXMLFile(tesseract_common::getTempPath() + ""
-                                                                                                                  "desc"
-                                                                                                                  "arte"
-                                                                                                                  "s_"
-                                                                                                                  "defa"
-                                                                                                                  "ult_"
-                                                                                                                  "plan"
-                                                                                                                  "_exa"
-                                                                                                                  "mple"
-                                                                                                                  "_inp"
-                                                                                                                  "ut."
-                                                                                                                  "xm"
-                                                                                                                  "l");
+  DescartesDefaultPlanProfile<double>::Ptr imported_plan_profile =
+      descartesPlanFromXMLFile(tesseract_common::getTempPath() + ""
+                                                                 "desc"
+                                                                 "arte"
+                                                                 "s_"
+                                                                 "defa"
+                                                                 "ult_"
+                                                                 "plan"
+                                                                 "_exa"
+                                                                 "mple"
+                                                                 "_inp"
+                                                                 "ut."
+                                                                 "xm"
+                                                                 "l");
```

Comment:
Same here. Is this the new clang format version?

Reply:
Clang format is doing this, but it is not as bad as before.